src/SFA.DAS.Employer.PR.Web/Models/Requests/EmployerAccountCreationViewModel.cs

```diff
@@ -40,11 +40,24 @@ public List<string> OperationDescriptions
             return operationDescriptions;
         }
     }
+
+    public string? ChangeNameLink { get; set; }
 }
 
-public class EmployerAccountCreationSubmitModel
+public class EmployerAccountCreationSubmitModel : EmployerAccountNamesModel
 {
     public required bool HasAcceptedTerms { get; set; }
+}
+
+public class EmployerAccountNamesModel : EmployerAccountNamesSubmitModel
+{
+    public string? BackLink { get; set; }
```

Comment:
Can we not use browser back here?

Reply:
I got it working, have removed all references to a BackLink field